src/components/AccountSecurity/MyOrganization.tsx

```diff
@@ -279,9 +279,11 @@ class MyOrganization extends Component<Props, State> {
       }),
     }).then((response) => response.json())
       .then((responseJSON) => {
-        const responseObject = responseJSON;
+        const responseObject = JSON.parse(responseJSON);
         const { status } = responseObject;
+
         if (status === 'SUCCESS') {
+          console.log('it was a success');
```

Comment:
Can you please remove the consol.log statements?

Reply:
theoretically logging is turned off on production so they cant actually see it but still we should remove all console log statements if possible.